provisioning/device/src/Models/SymmetricKeyRegistrationResult.cs

```diff
@@ -1,13 +1,19 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using Newtonsoft.Json;
+
 namespace Microsoft.Azure.Devices.Provisioning.Client
 {
     /// <summary>
-    /// Symmetric Key registration result.
+    /// Registration result returned when using symmetric key attestation.
     /// </summary>
     internal class SymmetricKeyRegistrationResult
     {
-        // this class has no implementation. does the service not send a payload?
+        /// <summary>
+        /// The Id of the enrollment group.
+        /// </summary>
+        [JsonProperty(PropertyName = "enrollmentGroupId")]
+        public string EnrollmentGroupId { get; internal set; }
```

Comment:
Okay, added, but we don't do anything with it. Does it need to be hooked up somewhere?

Reply:
FWIW, neither is `X509RegistrationResult.EnrollmentGroupId`. We deserialize it but don't do anything with it.

Should we?

Reply:
Looks like we haven't been exposing all fields in the public facing `DeviceRegistrationResult` (eg. `TpmRegistrationResult`, `X509RegistrationResult` etc.). Adding those in the public facing registration result class should fix it.

Reply:
Should we make them public get then?